src/backend/distributed/transaction/transaction_management.c

```diff
@@ -46,8 +46,18 @@ XactModificationType XactModificationLevel = XACT_MODIFICATION_NONE;
 /* list of connections that are part of the current coordinated transaction */
 dlist_head InProgressTransactions = DLIST_STATIC_INIT(InProgressTransactions);
 
-/* stack of active sub-transactions */
-static List *activeSubXacts = NIL;
+/* string to keep track of SET LOCAL statements executed within current subxact */
+/* set to NULL when pushing into new subxact or ending top xact */
```

Comment:
slightly unusual comment formatting

Reply:
I think I added one line in one commit and the other in another; I'll combine.